boards/arduino-mega2560/examples/mega2560-serial.rs

```diff
@@ -0,0 +1,32 @@
+#![no_std]
+#![no_main]
+
+extern crate panic_halt;
+use arduino_mega2560::prelude::*;
+
+#[arduino_mega2560::entry]
+fn main() -> ! {
+    let dp = arduino_mega2560::Peripherals::take().unwrap();
+
+    let mut porte = dp.PORTE.split();
+    let mut serial = arduino_mega2560::Serial::new(
+        dp.USART0,
+        porte.pe0,
+        porte.pe1.into_output(&mut porte.ddr),
+        57600,
+    );
+
+    // The following would also work, but needs +600% more bytes
+    // writeln!(serial, "Hello from Arduino!\r").unwrap();
+    serial.write_str("Hello from Arduino!\r\n").void_unwrap();
```

Comment:
The idiomatic way for writing to serial is to use `ufmt::uwriteln!()`.  I suspect you got this from another place in the code-base which wasn't updated yet?

```suggestion
    ufmt::uwriteln!(&mut serial, "Hello from Arduino!\r").void_unwrap();
```

Reply:
Done.  I believe the code was copied from the corresponding BigAVR6 example.

Reply:
Thanks! I'll try to remember I have to update that one as well.